CMakeLists.txt

```diff
@@ -143,7 +143,11 @@ if(WIN32)
 		# Suppress warnings
 		add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 		add_definitions(-D_SCL_SECURE_NO_WARNINGS)
-		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj /wd4251")
+		# 4003: not enough actual parameters for macro 'identifier'
```

Comment:
The macro causing the warning is this when no parameter is given (because the function returns void):

`#define RETURN_IF_NOT_BATTLE(X) if(!duringBattle()) {logGlobal->error("%s called when no battle!", __FUNCTION__); return X; }`

I fixed it by changing the macro parameter to "..."